src/components/DateRangePicker.js

```diff
@@ -230,10 +226,7 @@ class DateRangePicker extends React.Component {
 
   _handleScrimClick = () => {
     this.props.onClose();
```

Comment:
`_handleScrimClick` is now used for more than just the scrim element so the name doesn't fit anymore. Also, `_resetToPropValuesAndClose` is only used in that one function so I think the two functions could be combined. 

What do you think about moving this line in to `_resetToPropValuesAndClose` and getting rid of `_handleScrimClick`?

Reply:
I agree with that.  I'll update.